lib/AST/FrontendSourceFileDepGraphFactory.cpp

```diff
@@ -406,26 +406,19 @@ class UsedDeclEnumerator {
   StringRef swiftDeps;
 
   /// Cache these for efficiency
-  const DependencyKey sourceFileInterface;
```

Comment:
I see... looks like this wasn't even used. Is that right?

Reply:
Yup.